examples/sparse_tensor_partial_exchange_example/server.py

```diff
@@ -0,0 +1,116 @@
+import argparse
+from functools import partial
+from typing import Any, Dict, Optional
+
+import flwr as fl
+import torch.nn as nn
+from flwr.common.parameter import ndarrays_to_parameters
+from flwr.common.typing import Config, Parameters
+from flwr.server.client_manager import SimpleClientManager
+
+from examples.models.cnn_model import Net
+from examples.simple_metric_aggregation import evaluate_metrics_aggregation_fn, fit_metrics_aggregation_fn
+from examples.utils.functions import make_dict_with_epochs_or_steps
+from fl4health.server.base_server import FlServer
+from fl4health.strategies.fedavg_sparse_coo_tensor import FedAvgSparseCooTensor
+from fl4health.utils.config import load_config
+
+
+def get_initial_model_parameters(model: nn.Module) -> Parameters:
+    # Initializing the model parameters on the server side.
+    return ndarrays_to_parameters([val.cpu().numpy() for _, val in model.state_dict().items()])
+
+
+def fit_config(
+    batch_size: int,
+    sparsity_level: float,
+    current_server_round: int,
+    local_epochs: Optional[int] = None,
+    local_steps: Optional[int] = None,
+) -> Config:
+    config: Config = {
+        **make_dict_with_epochs_or_steps(local_epochs, local_steps),
+        "batch_size": batch_size,
+        "sparsity_level": sparsity_level,
+        "current_server_round": current_server_round,
+    }
+    config["fl_fit_round"] = True
+    return config
+
+
+def eval_config(
+    batch_size: int,
+    sparsity_level: float,
+    current_server_round: int,
+    local_epochs: Optional[int] = None,
+    local_steps: Optional[int] = None,
+) -> Config:
+    config: Config = {
+        **make_dict_with_epochs_or_steps(local_epochs, local_steps),
+        "batch_size": batch_size,
+        "sparsity_level": sparsity_level,
+        "current_server_round": current_server_round,
+    }
+    config["fl_fit_round"] = False
```

Comment:
This config parameter is used to inform the client whether the current round is an eval or fit round so it can decide which parameter exchanger to use.

Reply:
With the changes we discussed on zoom to the get_parameters function in `basic_client`, leveraging def fit or def eval to send this boolean rather than the configs. We should be able to unify these config generator functions 🙂